napari/viewer.py

```diff
@@ -5,11 +5,13 @@
 from qtpy.QtGui import QIcon
 from qtpy.QtWidgets import QApplication
 
+from ._qt.qt_event_timing import convert_app_for_timing
 from ._qt.qt_main_window import Window
 from ._qt.qt_viewer import QtViewer
 from ._qt.threading import wait_for_workers_to_quit, create_worker
 from .components import ViewerModel
 from . import __version__
+from .utils import perf
```

Comment:
same questions about lazy imports in `event_loop.py` can we avoid importing any of that machinery if we don't need it?

Reply:
I made used env var and made the import lazy here too.